policyengine_us/variables/gov/states/dc/tax/income/credits/eitc/dc_eitc_without_qualifying_child.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class dc_eitc_without_qualifying_child(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "DC EITC Without Qualifying Child Amount"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://code.dccouncil.gov/us/dc/council/code/sections/47-1806.04"  # (f)
+    defined_for = StateCode.DC
+
+    def formula(tax_unit, period, parameters):
+        employment_income = tax_unit("employment_income", period)
```

Comment:
To capture self-employment income as well, also as `employment_income` is person- rather than tax-unit-level
```suggestion
        earnings = tax_unit("tax_unit_earned_income", period)
```

Reply:
changed to tax_unit_earned_income